lua/lspconfig/util.lua

```diff
@@ -222,7 +222,7 @@ end)()
 
 -- Returns a function(root_dir), which, when called with a root_dir it hasn't
 -- seen before, will call make_config(root_dir) and start a new client.
-function M.server_per_root_dir_manager(_make_config)
+function M.server_per_root_dir_manager(make_config)
```

Comment:
@teto maybe you can shed some light on why this was prefixed an underscore in 6b8093c1f18f10a40121ab60957985cbf8da1987?

Reply:
dunno, maybe it conflicted, maybe I did similar to what was done with _root_dir. No issue changing it.